src/components/Restorans/RestoranTopFood.jsx

```diff
@@ -93,13 +120,111 @@ const RestoranTopFood = () => {
           </p>
           <div className='location'>
             <p>Restoran Top Food 021</p>
-            <p>Svetozala Miletica 26, 21000 Novi Sad</p>
-            <p>0644226471</p>
+            <div className='location-restoran'>
+              <img src={Location} alt='' />
+              <p>Svetozala Miletica 26, 21000 Novi Sad</p>
+            </div>
+            <div className='phone-restoran'>
+              <img src={Phone} alt='' className='phone-image' />
+              <p>0644226471</p>
+            </div>
             <img src={map} className='image-map' alt='' />{' '}
             {/*Stavljena slika samo zbog pregleda*/}
           </div>
         </div>
+        <div className='main-list-picture'>
+          <div className='list-picture1'>
+            <div className='first-box-picture'>
+              <div className='foreach-box'>
+                <img src={listpicture} alt='' />
+                <p>Lorem Ipsum</p>
+                <div className='title-rating'>
+                  <span className='title-list'>RESTORANI * Lorem ipsum</span>
+                  <span className='list-rating'>
+                    4.2
+                    <RiStarSFill />
+                  </span>
+                </div>
+              </div>
+              <div className='foreach-box'>
```

Comment:
You have this div with class name `foreach-box `keep repeating over and over again, please create array and render it with map function